article/sections/_9_conclusion.tex

```diff
@@ -40,3 +40,15 @@ \section{Conclusion}
 We then show that while the deconfounder algorithm might be a promising step in the right direction, 
 it still has some notable deficiencies. More specifically, we show that the deconfounder algorithm shows 
 to be sensitive to small errors in one's data.
+
+We hope that this chapter has provided enough preliminary evidence to demand modelers on the importance of using DAGs in their analyses.
+Using DAGs will allow modelers to clearly outline their assumptions about their own problems and will allow them to track their analysis workflow more easily.
+By now, choice modelers should have preliminary knowledge (and additional references) on how to:
+\begin{itemize}
```

Comment:
Can you change `DAG` in the bullets below to a more generic term like causal graph or causal diagram?
Sometimes undirected or cyclic causal graphs are most appropriate for one's system/problem.

Reply:
See #174

Reply:
One quick question. Did we say anywhere in the doc that we're using causal diagrams and causal graphs interchangeably?

Reply:
@bouzaghrane I don't think we ever spoke about it. I totally assumed the two terms as interchangeable given Pearl's 1995 paper being titled "Causal diagrams for empirical research". 😝

Reply:
@bouzaghrane this is what Joan and I had to say about the names in our 2018 paper:
> The figures most frequently used for these graphical displays are directed, acyclic graphs.
When used to encode causal assumptions, such graphs are known as “structural equation models” in transportation, econometrics, psychology, and sociology (Golob, 2003; Bollen and Pearl, 2013); 
“causal flow diagrams” or “system maps” in systems dynamics (Abbas and Bell, 1994; Shepherd, 2014);
“causal diagrams” in computer science and systems dynamics (Pearl, 2009b; Abbas and Bell, 1994); 
“influence diagrams” in statistics (Dawid, 2015);
and “causal graphs” or “path diagrams” in the social sciences (Morgan and Winship, 2015).

Reply:
I'm with you on that @timothyb0912 . I just didn't know if you wanted to stick with one term here as opposed to a mix of the two.

Reply:
@bouzaghrane oh for sure. I think it's a good idea to be consistent and stick to one. Thanks for pointing it out!